django/tasks/base.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.module_loading import import_string
+
+from .exceptions import InvalidTaskBackendError
+
+DEFAULT_TASK_ALIAS = 'default'
+
+# backends cache to make sure we don't create one per connection
+# all the backends should be thread-safe
+backends = {}
+
+
+def _create_task_backend(alias):
+    try:
+        conf = settings.QUEUES[alias]
+    except KeyError:
+        if alias != DEFAULT_TASK_ALIAS:
```

Comment:
`fallback_to_default` is imho not a good option since this is typically called internally (backend being set on a task). We'd either have to propagate the setting from `Task` to `get_backend` or use what @apollo13 and define `ALIASES` or something in `settings`.